libraries/chain/include/eosio/chain/exceptions.hpp

```diff
@@ -524,9 +524,11 @@ namespace eosio { namespace chain {
                                     3015016, "ABI has an unsupported version" )
       FC_DECLARE_DERIVED_EXCEPTION( duplicate_abi_action_results_def_exception,  abi_exception,
                                     3015017, "Duplicate action results definition in the ABI" )
+      FC_DECLARE_DERIVED_EXCEPTION(duplicate_abi_kv_table_def_exception, abi_exception,
+                                   3015018, "Duplicate kv_table definition in the ABI")
 
-   FC_DECLARE_DERIVED_EXCEPTION( contract_exception,           chain_exception,
-                                 3160000, "Contract exception" )
+      FC_DECLARE_DERIVED_EXCEPTION(contract_exception, chain_exception,
+                                   3160000, "Contract exception")
```

Comment:
The indentation here should be restored to what it was.

Reply:
restored the indentation